include/libdnf/base/transaction.hpp

```diff
@@ -79,6 +79,9 @@ class Transaction {
     // TODO(jrohel): Return reference instead of copy?
     std::vector<libdnf::base::TransactionPackage> get_transaction_packages() const;
 
+    /// @return the number of transaction packages.
+    std::size_t get_num_of_transaction_packages() const;
```

Comment:
Just a note / minor thing: if we would use the reference return value for `get_transaction_packages()` then this won't be needed I guess. 

Also I'd prefer not using abbreviations in the method name as it will be on public API, f.e. like `get_transaction_packages_count()`, what do you think?

Reply:
>Just a note / minor thing: if we would use the reference return value for get_transaction_packages() then this won't be needed I guess.

Yes. I'd rather return a constant reference instead of an expensive copy. The user can then make a copy only when needed. But here is some background. We should discuss. But not as part of this PR.

Thanks. `get_transaction_packages_count()` is exactly the name I was looking for.